deprecation.py

```diff
@@ -245,7 +269,9 @@ def _inner(*args, **kwargs):
                               stacklevel=2)
 
             return function(*args, **kwargs)
+
```

Comment:
We don't want to alter the spacing of the original code, here or below.

Reply:
I have returned the spacing to how it was originally, I believe pycharm did some auto-formatting (disabled now)